src/script/conversation/ConversationLabelRepository.ts

```diff
@@ -219,8 +225,12 @@ export class ConversationLabelRepository extends TypedEventTarget<{type: 'conver
 
   readonly removeConversationFromLabel = (label: ConversationLabel, removeConversation: Conversation): void => {
     label.conversations(label.conversations().filter(conversation => conversation !== removeConversation));
+    // Add conversations in a folder are deleted so folder must be deleted too
```

Comment:
I think you meant "All conversations" not "Add" :D 

```suggestion
    // Delete folder if it no longer contains any conversation
```